EXAMPLE.md

```
@@ -0,0 +1,133 @@
+
+- Add docs to describe the release integration (#6)
+- Add publish docs
+
+## [0.1.0] - 2025-06-13
+
+### Miscellaneous Tasks
+
+- Filter unwanted tags
+- Don't force conventional commits, not everything is going to have respected that historically
+- Set up tooling and git-cliff integration (#1)
+  - * chore: Set up tooling
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)